### ISSUE TITLE
Too much time spent computing conrey angles on some CMF pages

### ISSUE THREAD

**Issue by AndrewVSutherland** [LMFDB/lmfdb#5513]
We are occasionally getting timeouts on pages like https://www.lmfdb.org/ModularForm/GL2/Q/holomorphic/983/2/c/a/2/1/, where most of the time is being spent in line 517--520 of `web_neform.py`, which are executed 490*167 =  81830 times:
```
                        c = chi.conreyangle(p) * self.char_order
                        angle = float(c / self.char_order)
                        value = CDF(0,2*CDF.pi()*angle).exp()
                        self.character_values[p].append((angle, value))
```
This could surely be sped up (e.g. by performing just one call to Pari/GP per character or per prime), but it's still going to take a non-trivial amount of time, and most of the data being computed is not displayed unless the user asks for it, since by default only data for p < 50 is shown.  I think we should consider either doing this computation asynchronously (using an ajax call), or turning the "250 1000" links into URLs so the computation would only be done when the user asks for it.

In any case, something needs to be done.

**Comment by AndrewVSutherland** [comment]
Actually a better solution would be to only compute angle data for the character that is actually relevant on embedded newform pages like the one above, rather than computing data for all 490 characters in the orbit. I don't think this angle data is needed on the main newform orbit page [https://www.lmfdb.org/ModularForm/GL2/Q/holomorphic/983/2/c/a](https://www.lmfdb.org/ModularForm/GL2/Q/holomorphic/983/2/c/a).

**Comment by AndrewVSutherland** [comment]
This seems to have been solved by #5591
